airflow/models.py

```diff
@@ -4355,10 +4355,8 @@ def get_run(session, dag_id, execution_date):
 
     @property
     def is_backfill(self):
-        if "backfill" in self.run_id:
-            return True
-
-        return False
+        from airflow.jobs import BackfillJob
```

Comment:
Not a big deal: see https://stackoverflow.com/questions/19077381/what-happens-when-i-import-module-twice-in-python

It's imported like this in other places in this file, probably to avoid import cycles